monitor/crawler.py

```diff
@@ -53,30 +66,29 @@ def init_db_tables(self):
     def clear(self, metadata: bool = True, certificates: bool = True) -> None:
         if metadata is True:
             with self.db_conn:
-                # TODO: do we need to clear the states table here?
+                # TODO Clear the states table here?
                 for table in [self.STATE_DB_NAME, self.TEACHER_DB_NAME]:
                     self.db_conn.execute(f"DELETE FROM {table}")
 
         super().clear(metadata=metadata, certificates=certificates)
 
-    def store_state_metadata(self, state):
-        self.__write_state_metadata(state)
-
-    def __write_state_metadata(self, state):
-        from nucypher.network.nodes import FleetStateTracker
-        state_dict = FleetStateTracker.abridged_state_details(state)
-        # convert updated timestamp format for supported sqlite3 sorting
-        state_dict['updated'] = state.updated.rfc3339()
-        db_row = (state_dict['nickname'], state_dict['symbol'], state_dict['color_hex'],
-                  state_dict['color_name'], state_dict['updated'])
+    def store_state_metadata(self, state: dict):
+        # TODO Limit the size of this table - no reason to store really old state values
+        # TODO: Convert updated timestamp format for supported sqlite3 sorting
+        # state['updated'] = state['updated'].rfc3339()
```

Comment:
@KPrasch , this adjustment of `updated` format was how the previous states got properly sorted. SQLite allows for sorting based on the rfc3339 timestamp format.  See the query for previous states at https://github.com/nucypher/nucypher-monitor/blob/a7d2c3c6eaa4b4c3eb5efca9b209f4a03597db29/monitor/db.py#L47. The use of `ORDER BY datetime(updated) DESC LIMIT {limit}` was how the states were returned in order from newest to oldest and displayed right (newest) to left (oldest)

Reply:
Alternatively, if not used, we'd have to sort the result of the query ourselves.

Reply:
Addressed in `derekpierre/prev-states` branch